packages/hydrooj/src/service/server.ts

```
@@ -114,10 +115,9 @@
 
 export interface ConnectionHandler {
     domain: DomainDoc;
-    ctx: Context;
 }
-export class ConnectionHandler extends ConnectionHandlerOriginal {
-    constructor(_, ctx: Context) {
+export class ConnectionHandler extends ConnectionHandlerOriginal<Context> {
```

Biome (1.9.4)

[error] This class is unsafely merged with an interface.

The interface is declared here.

The TypeScript compiler doesn't check whether properties defined in the interface are initialized in the class.

(lint/suspicious/noUnsafeDeclarationMerging)